### PR TITLE
en, zh: explain TidbCluster spec.discovery (#1194)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #1194 to release-1.1
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In docs-tidb-operator repo:
git pr https://github.com/pingcap/docs-tidb-operator/pull/1203
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/docs-tidb-operator.git pr/1203:release-1.1-ae227f6bdb42
```

---

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

 explan TidbCluster `spec.discovery`

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [*] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [*] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -164,6 +164,34 @@ TiDB Operator 支持为 PD、TiDB、TiKV 挂载多块 PV，可以用于不同用
...
+
+`spec.discovery` 配置示例：
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v5.0.1
```
Comment (qiancai) [inline zh/configure-a-tidb-cluster.md:179]:
```suggestion
  version: v4.0.10
```

File: en/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -170,6 +170,34 @@ To enable `HostNetwork` for all supported components, configure `spec.hostNetwor
...
+
+A `spec.discovery` configuration example is as follows:
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v5.0.1
```
Comment (qiancai) [inline en/configure-a-tidb-cluster.md:185]:
```suggestion
  version: v4.0.10
```

**Comment by qiancai** [comment]
Hi @better0332, for now, could we still use v4.0.10 in v1.1 to align with other v1.1 documents?

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**